internal/googlereader/handler.go

```diff
@@ -49,6 +49,7 @@ func Serve(router *mux.Router, store *storage.Storage) {
 	sr := router.PathPrefix("/reader/api/0").Subrouter()
 	sr.Use(middleware.handleCORS)
 	sr.Use(middleware.apiKeyAuth)
+	sr.Use(middleware.maybeUnauthorizedGoogleReader)
```

Comment:
The middleware is not executed for the `/accounts/ClientLogin` endpoint defined above. Bad actors can still try to brute force username/passwords.